nixos/modules/services/cluster/kubernetes/default.nix

```diff
@@ -1018,9 +1018,9 @@ in {
             ${if (cfg.controllerManager.rootCaFile!=null)
               then "--root-ca-file=${cfg.controllerManager.rootCaFile}"
               else "--root-ca-file=/var/run/kubernetes/apiserver.crt"} \
+            ${optionalString (cfg.clusterCidr==null) "--allocate-node-cidrs=false"} \
```

Comment:
I think `if then else` would be more explicit and readable.

Reply:
Done